src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -0,0 +1,223 @@
+            </p>
+          </div>
+          <div className="govuk-grid-column-one-third">
+            <RelatedInformation heading="Related information">
+              <ul className="govuk-list">
+                <li>
+                  <a
+                    href="https://www.gov.uk/government/organisations/ofsted/about/statistics"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Ofsted statistics
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://www.education-ni.gov.uk/topics/statistics-and-research/statistics"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Educational statistics for Northern Ireland
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://www.gov.scot/statistics-and-research/?cat=filter&amp;topics=Education"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Educational statistics for Scotland
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://statswales.gov.wales/Catalogue/Education-and-Skills"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Educational statistics for Wales
+                  </a>
+                </li>
+              </ul>
+            </RelatedInformation>
+          </div>
+        </div>
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-third">
+            Search and filters here
+          </div>
+          <div className="govuk-grid-column-two-thirds">
+            <h2
+              className="govuk-!-margin-bottom-2"
+              aria-live="polite"
+              aria-atomic="true"
+            >
+              {totalResults} {totalResults !== 1 ? 'results' : 'result'}
+            </h2>
+
+            <a href="#searchResults" className="govuk-skip-link ">
+              Skip to search results
+            </a>
+            <p
+              aria-live="polite"
+              aria-atomic="true"
+              className="govuk-!-margin-top-1"
+            >{`Page ${currentPage} of ${paging.totalPages}, showing all publications`}</p>
+
+            {/* TODO EES-3517
+             <p className="govuk-visually-hidden">
+              Sorted by newest publications
+            </p>
+            <p>Current filters here</p>
+            <p>Sorting controls here</p>*/}
+
+            <hr />
+
+            <div
+              className="govuk-visually-hidden"
+              id="searchResults"
+              ref={resultsRef}
+              tabIndex={-1}
+            >
+              Search results
+            </div>
+
```

Comment:
We'll want to show some loading spinner whenever the page (and sort, filters, etc) are changed